e2e/cypress/support/commands.ts

```diff
@@ -347,3 +316,60 @@ function waitUntilAdminTokenPermissioned(attempts: number): void {
   });
   }
 }
+
+Cypress.Commands.add('waitForClientRunsNode', (nodeId: string, maxRetries: number) => {
```

Comment:
can you move these above the helpers section (line 307)?

Reply:
Thanks.